modular_skyrat/modules/GAGS/greyscale_configs.dm

```diff
@@ -1413,6 +1423,16 @@ digi
 	name = "CIN Surplus Chestrig (OBJ)"
 	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
 
+// Bigrig
+/datum/greyscale_config/cin_surplus_bigrig
+	name = "CIN Large Surplus Chestrig"
+	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/nri_surplus_equipment/bigrig.json'
+
+/datum/greyscale_config/cin_surplus_bigrig/object
+	name = "CIN Large Surplus Chestrig (OBJ)"
+	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
```

Comment:
Usually greyscale configs are the other way around. Base type is the obj, subtype is /worn and (Worn). There a reason these aren't?

Reply:
That's just what I did for the other greyscale cin stuff and im not about to change it now